detox/package.json

```diff
@@ -5,44 +5,39 @@
   "author": "Mattermost, Inc.",
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "7.18.6",
-    "@babel/plugin-transform-modules-commonjs": "7.22.5",
-    "@babel/plugin-transform-runtime": "7.22.5",
-    "@babel/preset-env": "7.22.5",
-    "@jest/test-sequencer": "29.5.0",
-    "@types/jest": "29.5.2",
-    "@types/tough-cookie": "4.0.2",
-    "@types/uuid": "9.0.2",
-    "aws-sdk": "2.1398.0",
-    "axios": "1.4.0",
-    "axios-cookiejar-support": "4.0.6",
-    "babel-jest": "29.5.0",
+    "@babel/plugin-transform-modules-commonjs": "7.22.15",
+    "@babel/plugin-transform-runtime": "7.22.15",
+    "@babel/preset-env": "7.22.20",
+    "@jest/test-sequencer": "29.7.0",
+    "@types/jest": "29.5.5",
+    "@types/tough-cookie": "4.0.3",
+    "@types/uuid": "9.0.4",
+    "aws-sdk": "2.1462.0",
+    "axios": "1.5.0",
+    "axios-cookiejar-support": "4.0.7",
+    "babel-jest": "29.7.0",
     "babel-plugin-module-resolver": "5.0.0",
     "client-oauth2": "4.3.3",
     "deepmerge": "4.3.1",
-    "detox": "20.9.1",
+    "detox": "20.11.4",
```

Comment:
we should match this version in the root packages.json as well

Reply:
Got it, now added in the recent commits